borg/archiver.py

```diff
@@ -628,38 +628,50 @@ def do_break_lock(self, args):
-        not specified. The style selector is `fm`. For the purpose of these patterns,
-        the path separator ('\\' for Windows and '/' on other systems) is not treated
-        specially. For a path to match a pattern, it must completely match from start
-        to end, or must match from the start to just before a path separator. Except
-        for the root path, paths will never end in the path separator when matching
-        is attempted. Thus, if a given pattern ends in a path separator, a '*' is
-        appended before matching is attempted.
-
-        Regular expressions similar to those found in Perl are supported with the
-        selection prefix `re:`. Unlike shell patterns regular expressions are not
-        required to match the complete path and any substring match is sufficient. It
-        is strongly recommended to anchor patterns to the start ('^'), to the end
-        ('$') or both. Path separators ('\\' for Windows and '/' on other systems) in
-        paths are always normalized to a forward slash ('/') before applying
-        a pattern. The regular expression syntax is described in the `Python
-        documentation for the re module
-        <https://docs.python.org/3/library/re.html>`_.
-
-        Prefix path patterns can be selected with the prefix `pp:`. This pattern
-        style is useful to match whole sub-directories. The pattern `pp:/data/bar`
-        matches `/data/bar` and everything therein.
+        `Fnmatch <https://docs.python.org/3/library/fnmatch.html>`_, selector `fm:`
+
+            These patterns use a variant of shell pattern syntax, with '*' matching
+            any number of characters, '?' matching any single character, '[...]'
+            matching any single character specified, including ranges, and '[!...]'
+            matching any character not specified. For the purpose of these patterns,
+            the path separator ('\\' for Windows and '/' on other systems) is not
+            treated specially. Wrap meta-characters in brackets for a literal match
+            (i.e. `[?]` to match the literal character `?`). For a path to match
+            a pattern, it must completely match from start to end, or must match from
+            the start to just before a path separator. Except for the root path,
+            paths will never end in the path separator when matching is attempted.
+            Thus, if a given pattern ends in a path separator, a '*' is appended
+            before matching is attempted.
+
+        Shell-style patterns, selector `sh:`
+
+            Like fnmatch patterns these are similar to shell patterns. The difference
+            is that the pattern may include `**/` for matching zero or more directory
+            levels, `*` for matching zero or more arbitrary characters with the
+            exception of any path separator.
+
+        Regular expressions, selector `re:`
+
+            Regular expressions similar to those found in Perl are supported. Unlike
```

Comment:
This sentence was in there before and is a quote from the [re module documentation](https://docs.python.org/3/library/re.html#module-re).